README.md

```diff
@@ -117,9 +112,7 @@ After you're finished please remove all the comments and instructions!
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
```

Comment:
- [ ] As requested by the previous reviewer, kindly fill in all the necessary sub sections under the getting started section. The prerequisites are those things a user will reuwire before they can clone your project. You add a comment to clone under the `clone this repo with:` etc.